### PR TITLE
{devel}[GCCcore/7.2.0] LCov 1.13

### PR DESCRIPTION
**Pull request by rubendibattista**

### CODE REVIEW

File: easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+name = 'LCov'
+version = '1.13'
+
+homepage = 'http://ltp.sourceforge.net/coverage/lcov.php'
+description = "LCOV - the LTP GCOV extension"
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+source_urls = ["http://downloads.sourceforge.net/ltp/"]
```
Comment (boegel) [inline easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb]: @rubendibattista Please use `SOURCEFORGE_SOURCE` here (which uses `https` now thanks to the changes in https://github.com/easybuilders/easybuild-framework/pull/2648)

**Comment by rubendibattista** [comment]
Anything else needed?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2043.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/01439b193811947fb1699567ab1031f0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (1 easyconfigs in this PR)
node3156.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c091f6413bf1c7c94cf38857904ba8c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (1 easyconfigs in this PR)
node2616.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fe7c9e3b660e1b01baa635cd543f6d1b for a full test report.

**Comment by boegel** [comment]
@rubendibattista It looks like `44972c878482cc06a05fe78eaa3645cbfcbad6634615c3309858b207965d8a23` was actually correct...

The download failed for me:

```
$ file data/EasyBuild/sources/l/LCov/lcov-1.13.tar.gz
data/EasyBuild/sources/l/LCov/lcov-1.13.tar.gz: HTML document, UTF-8 Unicode text, with very long lines
```

So it seems like the switch to `SOURCEFORGE_SOURCE` wasn't so wise after all...?

File: easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+name = 'LCov'
+version = '1.13'
+
+homepage = 'http://ltp.sourceforge.net/coverage/lcov.php'
+description = "LCOV - the LTP GCOV extension"
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
```
Comment (boegel) [inline easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb]: @rubendibattista You'll need to switch this back to `http://downloads.sourceforge.net/ltp/`, the `/ltp` is probably preventing a correct download via the `SOURCEFORGE_SOURCE` template...

Comment (verdurin) [reply easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb]: @boegel - @rubendibattista fixed it by using Github instead, which works fine in my testing.

**Comment by verdurin** [comment]
Going in, thanks @rubendibattista!

**Pull request merged**